frameworks/g-cloud-9/questions/services/multiqSecurityGovernance.yml

```diff
@@ -0,0 +1,15 @@
+name: Security governance
+question: Security governance
+
+depends:
+  - "on": lot
+    being:
+      - cloud-hosting
+      - cloud-software
+
+type: multiquestion
+questions:
+  - securityGovernanceAccreditation
+  - securityGovernanceStandards
+  - otherSecurityGovernanceStandards
```

Comment:
Same as before, have tried to keep filenames the same as the question name/heading. This seems like a good argument against doing that, but maybe let's discuss...